test/idtable.cc

```diff
@@ -1,4 +1,4 @@
-#include "mjolnir/idtable.h"
+#include "../src/mjolnir/idtable.h"
 
```

Comment:
You can also revert the changes in this file

Reply:
No, I can't; if I revert it, the header can't be found, and CI fails.

Reply:
Specifically:
```
valhalla/test/idtable.cc:1:10: fatal error: mjolnir/idtable.h: No such file or directory
    1 | #include "mjolnir/idtable.h"
```

Reply:
same as above, i didnt see you actually moved the header to the source directory. though as i mentioned before i wonder why it even makes sense to have this class as a separate header at this point

Reply:
> I wonder why it even makes sense to have this class as a separate header at this point

The short answer is that I'm not sure how the unit test is supposed to work otherwise. If you have suggestions...?